pulp_python/app/fields.py

```diff
@@ -1,12 +0,0 @@
-from drf_spectacular.utils import extend_schema_field
-from drf_spectacular.types import OpenApiTypes
-from rest_framework import serializers
-
-
-@extend_schema_field(OpenApiTypes.OBJECT)
-class JSONObjectField(serializers.JSONField):
```

Comment:
The reason this can be removed, I presume, because https://github.com/tfranzel/drf-spectacular/issues/1095 was fixed?

Reply:
See my messages in pulp-dev (https://matrix.to/#/!aVApiNMtnstWbwDcVU:matrix.org/$GRdEDvdyB04MXXbc66NWVNhCaI_VgdnhUb2xuJ1OJWs?via=matrix.org&via=ctrl-c.liu.se&via=x9c4.spdns.org), but the tldr is that with 3.70 we are using a new version of the bindings generator. This new version introduces pydantic models to check the types, so the mismatch between what we say the field is in the api and what we accept causes errors in the new bindings. This previous change was because the ruby-bindings broke when drf-spectular updated, so I have no idea if the ruby bindings will break again when removing this, but they should also be using the new bindings generator so hopefully they don't.

Reply:
Nice!